### ISSUE TITLE
The failure of reproducing results in the paper

### ISSUE THREAD

**Issue by cwhgn** [hasanirtiza/Pedestron#115]
I am trying to reproduce the results of 'CityPersons->CityPersons' in the paper Table.5/6, i.e. marked red boxes shown below. 
![image](https://user-images.githubusercontent.com/6065208/137901001-16f02687-37c9-4cb3-bad0-45a41496d791.png)
![image](https://user-images.githubusercontent.com/6065208/137901850-cd79e28c-2872-420b-86da-ef4fbeb03928.png)

Specifically, I directly run this command ("./tools/dist_train.sh configs/elephant/cityperson/cascade_hrnet.py 8") on 8 GPUs, and used the epoch_240 model for testing. However, the result of the trained model is only 'R/HO 15.55/40.86', which is much lower than the reported results 'R/HO 11.2/37.0' in the paper. Is there any specific setting I missed? Can you help me to reproduce the reported results in the paper. Many thanks~

**Comment by hasanirtiza** [comment]
Please first refer to this issue #51. In a nutshell, you need to run only till 20 epoch (240 is a typo), and since it is a validation set, you evaluate every checkpoint on the validation set. If you are interested in test performances, please check the leaderboards of [CityPerson dataset](https://github.com/cvgroup-njust/CityPersons), where we also report our performances.

**Comment by cwhgn** [comment]
Thanks so much, it works. I can reproduce the results within 20 epoch. Another question, do I also need to run only 20 epoch for "Caltech->Caltech"?

**Comment by hasanirtiza** [comment]
Yes, try 20, for Caltech, as I remember you might need even fewer epochs.

**Comment by cwhgn** [comment]
OK, Thanks!
